README.md

```diff
@@ -107,10 +107,57 @@ Stop the application by pressing `Ctrl + C`
 7. Start your API with `./mvnw clean spring-boot:run` (if it's already running, restart it) and confirm there are no errors
 8. Check your database contains an "ious" table with the correct columns and data types
 
+## Exercise 2
+
+1. Create an IOUService class that accepts an IOURepository as a dependency and implements the following methods:
+    - `List<IOU> getAllIOUs()`
+    - `IOU getIOU(UUID id) throws NoSuchElementException`
+    - `IOU createIOU(IOU iou) throws IllegalArgumentException, OptimisticLockingFailureException`
+    - `IOU updateIOU(UUID id, IOU updatedIOU) throws NoSuchElementException`
+    - `void deleteIOU(UUID id)`
+2. Create an `IOUController` class that implements the endpoints below. Ensure your service class is injected as a dependency and apply the appropriate annotations
```

Comment:
[Are these the endpoints that are to be implemented?](https://github.com/cbfacademy/java-rest-api-exercise/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R138-R144) 

If they are, could lines 138-144 be moved here so it's clearer

Reply:
Thanks for catching that, that's where they were supposed to be